src/component/missions/Missions.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import { useDispatch, useSelector } from 'react-redux';
+import { joinMission } from '../redux/missions/missionsSlice';
+import './mission.css';
+
+export default function Missions() {
+  const dispatch = useDispatch();
+  const isLoading = useSelector((state) => state.missions.isLoading);
+
+  const mission = useSelector((state) => state.missions.missions);
+
+  // function changeMemberState(id) {
+  // }
+
```

Comment:
- [ ] to keep your repo clean kindly remove all commented codes from your project 🙏